modules/claims_api/spec/fixtures/form_526_json_api.json

```diff
@@ -60,7 +60,7 @@
           },
           {
             "activeDutyEndDate": "1999-01-01",
-            "serviceBranch": "Air Force Reserves",
+            "serviceBranch": "Air Force",
```

Comment:
What was the reason for this needing to change? Wondering if it will cause some business logic around `Reserves` service period to no longer get triggered

Reply:
If we're updating to the only the values returned by the BGS service that EVSS is ostensibly validating from, `Air Force Reserves` isn't in that list

Reply:
ah shoot, i missed that. fair enough i guess. im surprised Reserves isn't in that list anywhere 🤷🏻